apps/storybook/panda.config.ts

```diff
@@ -1,14 +1,13 @@
 import { defineConfig } from "@pandacss/dev";
-import { buildPandaConfig } from "@tailor-platform/dev-config/pandacss";
+import { buildPandaConfig } from "../../packages/design-systems/src/pandacss";
```

Comment:
dev-configをつかわない(or使えない?)理由が知りたいです。

Reply:
design-systemsがeslintなどの設定をdev-configに依存しており、このpanda.configの設定はdesign-systemsのtokenやrecipesに依存しているためdev-config下においてしまうと循環参照エラーが出るようになってしまったからです。

これが以前まで発生していなかったのがちゃんとわかっていないのですが、発生するタイミングがapps/storybook以下でpanda codegenを行うタイミングなのでdesigns-sytemsを直接参照するのではなくnode_modules以下を参照するようにした結果かなと思っています。
どちらにしてもpanda.comfigをdev-configにおいた場合がdev-config単体で使えないのでそれならdesign-systemsからexportするほうがいいかなという判断です

Reply:
なるほど、理解です。たしかにpandacssのみdev-configにあってもというのは同意なので、design-systemsに配置でいきましょう。

Reply:
ここは以下でいいと思いましたが、動きませんか?

```suggestion
import { buildPandaConfig } from "@tailor-platform/design-systems/pandacss";
```

Reply:
"@tailor-platform/design-systems/pandacss";
これなんですが、storybookの起動には問題がないのですが、（vite.configで解決できるから）tsconfigがこの階層に効かないのか常にts-errorが出るようになるので↓にしています。
"../../packages/design-systems/src/pandacss";